radiography/src/main/java/radiography/ViewStateRenderers.kt

```diff
@@ -55,49 +56,60 @@ public object ViewStateRenderers {
   @JvmField
   public val DefaultsNoPii: List<ViewStateRenderer> = listOf(
       ViewRenderer,
-      textViewRenderer(includeTextViewText = false, textViewTextMaxLength = 0),
+      ComposeViewRenderer,
+      textViewRenderer(showTextValue = false, textValueMaxLength = 0),
       CheckableRenderer,
-  ) + ComposeLayoutRenderers.DefaultsNoPii
+  )
 
   @JvmField
   public val DefaultsIncludingPii: List<ViewStateRenderer> = listOf(
       ViewRenderer,
-      textViewRenderer(includeTextViewText = true),
+      ComposeViewRenderer,
+      textViewRenderer(showTextValue = true),
```

Comment:
I think maybe the verb shouldn't be `show`, because this controls what we concatenate to a string not what we show on a string.

How about `showTextValue` ?

Reply:
and we could probably shorten to `renderText` ? idk.

Reply:
https://github.com/square/radiography/issues/73

Reply:
I'm not sure I understand your distinction between "concatenate" and "show" in this context, but my LGAF is extremely low for this parameter. `includeTextViewText` just seemed unnecessarily long and verbose, and not strictly accurate anymore with the compose support. I'm fine with `renderText`.